vnext/Microsoft.ReactNative/Fabric/Composition/CompositionDynamicAutomationProvider.cpp

```diff
@@ -405,4 +412,70 @@ HRESULT __stdcall CompositionDynamicAutomationProvider::get_IsReadOnly(BOOL *pRe
   return S_OK;
 }
 
+HRESULT __stdcall CompositionDynamicAutomationProvider::SetValue([in] double val) {
+  auto strongView = m_view.view();
+
+  if (!strongView)
+    return UIA_E_ELEMENTNOTAVAILABLE;
+
+  strongView->setAcccessiblityRangeValue(val);
+  return S_OK;
+}
+
+HRESULT __stdcall CompositionDynamicAutomationProvider::get_LargeChange(double* pRetVal) {
+  if (pRetVal == nullptr)
+    return E_POINTER;
+
+
+
+  return S_OK;
```

Comment:
If you are not going to implement this, you should return quiet NaN:
https://learn.microsoft.com/en-us/windows/win32/api/uiautomationcore/nf-uiautomationcore-irangevalueprovider-get_largechange